apps/thunderhub/docker-compose.yml

```diff
@@ -19,6 +18,7 @@ services:
       ACCOUNT_CONFIG_PATH: "/data/thubConfig.yaml"
       YML_ENV_1: "$LND_IP:$LND_GRPC_PORT"
       TOR_PROXY_SERVER: "socks://$TOR_PROXY_IP:$TOR_PROXY_PORT"
+      MEMPOOL_URL: "http://${DEVICE_HOSTNAME}:$APP_MEMPOOL_PORT"
```

Comment:
`DEVICE_HOSTNAME` is not accessible from the compose file. You can use `APP_DOMAIN`. 
```suggestion
      MEMPOOL_URL: "http://$APP_DOMAIN:$APP_MEMPOOL_PORT"
```

But two issues remain: 
- the `mempool` app may not be installed 
- when `mempool` is installed, it looks like Thunderhub tries to proxy the request through Tor (and it obviously fails)
<img width="1077" alt="image" src="https://user-images.githubusercontent.com/38882571/162053590-2fd54e7d-f11f-4333-b5c2-fb89fed63559.png">

One possible way to fix this: fallback to the public Mempool.space instance if the local one is not available (should the user be warned @lukechilds?) and skip Tor for non-onion urls.

Pinging @apotdevin on this as well.

Reply:
My bad, didn't see the `mempool` dependency above. Anyway, Umbrel doesn't support inter-app dependencies yet. It might be better to remove `mempool` and go for the possible fix I described in my previous comment.

Thoughts @lukechilds?

Reply:
I personally love the mempool integration as it in line with umbrel's mission statement increases privacy.

Reply:
I love the Mempool integration but yeah I think it might be better to skip it until we have a better way to support it. 

Some issues that make it not too simple:
- We need to detect if the Mempool app is installed
- We need to know where the Mempool is exposed at
- We need to display the approprate Mempool address based on which address Thunderhub is being accessed over.

e.g If Thunderhub is being access via umbrel.local, we can just append the Mempool port, but if Thunderhub is being acccess over Tor, we need to link to Mempool's Tor address.